react/features/speaker-stats/components/native/SpeakerStatsButton.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import { connect } from 'react-redux';
+import type { Dispatch } from 'redux';
+
+import { createToolbarEvent, sendAnalytics } from '../../../analytics';
+import { translate } from '../../../base/i18n';
+import { IconPresentation } from '../../../base/icons';
+import { AbstractButton } from '../../../base/toolbox/components';
+import type { AbstractButtonProps } from '../../../base/toolbox/components';
+import { navigate } from '../../../conference/components/native/ConferenceNavigationContainerRef';
+import { screen } from '../../../conference/components/native/routes';
+
+type Props = AbstractButtonProps & {
+
+    /**
+     * True if the navigation bar should be visible.
+     */
+    dispatch: Dispatch<any>
+};
+
+
+/**
+ * Implementation of a button for opening speaker stats dialog.
+ */
+class SpeakerStatsButton extends AbstractButton<Props, *> {
+    accessibilityLabel = 'toolbar.accessibilityLabel.speakerStats';
```

Comment:
these are shared props between native and web implementation, you could create an AbstractSpeakerStatsButton for them, and have different implementation web / native for the _handleClick